openpype/hosts/maya/plugins/publish/extract_camera_alembic.py

```diff
@@ -61,10 +61,11 @@ def process(self, instance):
 
             if bake_to_worldspace:
                 job_str += ' -worldSpace'
-                for member in member_shapes:
-                    self.log.info(f"processing {member}")
+
+                for member in members:
                     transform = cmds.listRelatives(
-                        member, parent=True, fullPath=True)[0]
+                        member, parent=True, fullPath=True)
+                    transform = transform[0] if transform else member
```

Comment:
Not entirely sure but this is something to be aware of. Solely looking at this code if a camera instance previously contained a parent group of the camera instead of the camera itself the old logic would still select the 'camera' for export.

I suspect this will now change the behavior for older camera instances that originally would not include the parent hierarchy.

```
group1/group2/camera
```

Would previously export as
```
camera
```

I suspect after this change the export is now:
```
group1/group2/camera
```